### DISCUSSION TITLE
August Adventures: Exploring GitHub 🧭 [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#64873] [New to GitHub]
![Header of New to GitHub category check-in shows a purple starry background with shooting stars and clouds next to an octocat in a beret holding a stack of books.](https://github.com/community/community/assets/92824518/3ff6f15b-be36-4fac-a2c8-e1bc3ba92918)

You don’t have to travel far to discover the splendors of the GitHub Universe nor to meet some amazing community guides for your journey. Unpack the latest with us below: 

### 🌟Community Superstars: A round of applause for these community citizens 👏 
* @seniorcoder72: Has been helping all over the category complete with screenshots and a willingness to answer any other questions that come up. Check out their model community citizenship [here](https://github.com/orgs/community/discussions/63668), [here](https://github.com/orgs/community/discussions/63669), and [here](https://github.com/orgs/community/discussions/63670)
* @w8ste: Broke down their [instructions](https://github.com/orgs/community/discussions/63387) on how to change your default branch name in a super digestible way - really great stuff! 
* @antzim: There was a lot of back and forth and code to work through on a [post about cloning repository permissions](https://github.com/orgs/community/discussions/64199), but that didn’t stop @antzim from solving it. They shared their own set up and approached this quite curiously - nicely done. 
  
💫 Want to nominate a community member for a feature above? Just tag me         (@queenofcorgis) where you’ve seen great community behavior. 💫

### 📚 Rousing Reads and Beyond: GitHub-produced goodies 🍭
* **From the NFL to Software Engineer at GitHub**: @sgoldfarb2 shares her journey from working in video editing to coding for GitHub in her [blogpost](https://github.com/orgs/community/discussions/63383) 🏈.  
* Treat accessibility issues as bugs, not feature requests: Follow a Drupal’s (@drupal) lead, prioritize and systemically squash accessibility bugs :accessibility: . [Learn how to get started](https://github.com/readme/guides/fix-accessibility-bugs) with @mgifford. 
* **Nine years of the GitHub Security Bug Bounty program**: It was another record year for our Security Bug Bounty program! @thejillboss highlight some achievements we’ve made together with the bounty community in 2022 on the [blog](https://github.blog/2023-08-14-nine-years-of-the-github-security-bug-bounty-program/) 🐞. 

### 🪐Universe 2023: Resources to get you ready
* **Universe 2023 tickets are available now**: Tickets are available (with an Early-Bird discount!). [Get tickets and learn more about Universe](https://github.com/orgs/community/discussions/63834).
* **Build for today, transform for tomorrow at GitHub Universe 2023**: Read our [blogpost](https://github.blog/2023-08-15-build-for-today-transform-for-tomorrow-at-github-universe-2023/) to get even more excited about Universe 2023. 

Are you planning on attending Universe 2023? Let us know if you’re going to be joining us in person or virtually in the comments.

**Comment by Zack4nuuu** [comment]
I just signed up and plan on attending virtually!

**Reply by queenofcorgis** [reply]
Nice, we're glad to have you. Stay tuned for more Universe updates!

**Comment by queenofcorgis** [comment]
Welcome September with our latest [New to GitHub check-in](https://github.com/orgs/community/discussions/66384)!

**Comment by Yakutmuder9** [comment]
That is great!
